Unix/tools/auth_tests_cleanup.sh

```diff
@@ -2,12 +2,12 @@
 # 
 # output directory and its softlink target are the two arguments
 
-ntlm_file=$1/tmp/ntlm
+#ntlm_file=$1/etc/ntlm
 
-if [ -f $ntlm_file ]; then
-    echo "Removing NTLM file: $ntlm_file"
-    rm -f $ntlm_file
-fi
+#if [ -f $ntlm_file ]; then
+#    echo "Removing NTLM file: $ntlm_file"
+#    sudo rm -f $ntlm_file
+#fi
```

Comment:
I deleted the lines.  I was concerned there would be some complication requiring the ntlm creds file to be destroyed but I don't see any.   The only "vulnerability" would be on a dev machine.